### DISCUSSION TITLE
Rotary encoder sensibility

### DISCUSSION THREAD

**Discussion by Iorobyy11** [probonopd/MiniDexed#322] [Q&A]
Hi everyone
Question about rotary encoder : response sensibility  is low , for example when i scroll the Patch Voices.
It seem there is a late Between rotation and display act , Maibe something to adjust in encoder software interface sensibility…
I rotate a lot before (3-4 rotation click) to switch to another Voice.
How can i increase or adust sensibility ?
Thanks

**Comment by craigyjp** [comment]
Do you have 10nf capacitors across the A&B terminals to ground? Also maybe a capacitor from switch to ground?

These are recommended for encoders and the Pi.

**Reply by dhstsw** [reply]
A & B terminals?

**Reply by craigyjp** [reply]
A standard encoder has 3 pins on one side A, gnd, B and the switch on the other. So connect the caps across A to center and B to center.

**Reply by dhstsw** [reply]
Thanks.
I'm using a KY-040 just like the one in the diagrams:
![image](https://user-images.githubusercontent.com/59790194/183291926-ed7c8bce-013d-48a2-b16a-381e95fadeb5.png)

There are CLK, DT, the switch, + and gnd.
I assume A and B are CLK and DT?

thanks.

**Reply by craigyjp** [reply]
yes, if you look on the github site the A and B are CLK and DT.

**Reply by dhstsw** [reply]
Thanks

**Reply by dhstsw** [reply]
> yes, if you look on the github site the A and B are CLK and DT.

After more than a month i found the time to solder the 10nf capacitors (directly on the back of the encoder itself).
Now it works absolutely perfect.

Thanks!

**Reply by Iorobyy11** [reply]
You mean 2 capacitors 10nf - clk to ground and dt to ground?

**Reply by dhstsw** [reply]
Exactly.

**Reply by Iorobyy11** [reply]
👍 thanks

**Comment by fedekrum** [comment]
Hi. I am willing to buy capacitors, but I have no electronics experience (nor knowledge).
I have found 3 different capacitors from a seller:
3KV 10nf(103)
2KV 10nF (103)
1KV 10nF (103)

another seller sells this:
50V - 10NF

and another:
2J103J 10NF 630V

Not sure what to buy.
I would appreciate any advice.

**Reply by plamen-dzhorev** [reply]
Raspberry PI is powered by 5V only so you can use the smallest possible voltage - 50V/10nF. If you have pictures of the capacitors, just choose the most convenient one for you, all will do the job but these with higher voltage can be bigger in size.

**Reply by fedekrum** [reply]
thanks @plamen-dzhorev !!!

**Comment by b1tfl0w** [comment]
Hello, I have a question about this. Some time ago, I created a program for launching scripts on the Pi [https://github.com/b1tfl0w/pictrl](https://github.com/b1tfl0w/pictrl). It’s quite simple: it uses a KY-040 rotary encoder to select the script to run and an SSD1306 display to show the menu. I didn’t need to add capacitors, and it never misses a step while rotating.
So, I’m wondering why capacitors are required in minidexed but not in my project? Maybe the library is implemented differently?

**Reply by penfold42** [reply]
They're probably not required as of the changes a couple of weeks ago.
#1008
